realm-annotations-processor/src/main/java/io/realm/processor/Utils.java

```diff
@@ -93,6 +93,20 @@ public static String getGenericType(VariableElement field) {
         return genericType;
     }
 
+    /**
+     * Strips the package name from a fully qualified class name.
+     */
+    public static String stripPackage(String clazz) {
```

Comment:
`clazz` should only be used for actual classes, not strings. This should be `fullyQualifiedClassName` for clarity